test/test_autograd.py

```diff
@@ -1027,7 +1027,14 @@ def test_functional_blas(self):
         def compare(fn, *args):
             unpacked_args = tuple(arg.data if isinstance(arg, Variable) else arg
                                   for arg in args)
-            self.assertEqual(fn(*args).data, fn(*unpacked_args))
+            unpacked_result = fn(*unpacked_args)
+            packed_result = fn(*args).data
+            # if non-Variable torch function returns a scalar, compare to scalar
+            if not torch.is_tensor(unpacked_result):
+                assert(packed_result.dim() == 1)
+                assert(packed_result.nelement() == 1)
```

Comment:
can you remove parenthesis around assert conditions? There's this funny example :wink:
```python
assert(False,)
```